RAKE/RAKE.py

```diff
@@ -4,8 +4,11 @@
 # Automatic keyword extraction from indi-vidual documents.
 # In M. W. Berry and J. Kogan (Eds.), Text Mining: Applications and Theory.unknown: John Wiley and Sons, Ltd.
 
+from __future__ import absolute_import
 import re
 import operator
+import six
+from six.moves import range
```

Comment:
also why can't we use the standard range instead of importing the module, and why both import six and six.moves? Also range is a standard keyword so i have concerns importing another function as that name